README.md

```diff
@@ -15,8 +15,15 @@ If you'd like a step-by-step tutorial on this project, check out our YouTube vid
 ## 🛠️ Getting Set-Up
 
 
-This repository requires the creation of an `.env` file with the following variables to communicate with the lemon.markets API: `MIC`, `API_KEY` 
-`TRADING_URL` and `MARKET_URL`. It also needs a `BOT_TOKEN` to communicate with the Telegram API.
+This repository requires the creation of an `.env` file with the following variables to communicate with the lemon.markets API:
```

Comment:
Telling users to create the .env was something I didn't change from the original project actually. I think the reason it wasn't pre-created is because we have the gitignore preventing the .env from being pushed for security reasons (as .env contains API keys). We could pre-create the .env and remind users to add it to the gitignore maybe?

Also, adding BOT_TOKEN to the table is a good idea, I'll make that change to the README.

Reply:
Yes, adding it to .gitignore is a great idea! My concern is also that users may not fully know the format we wish to have of the .env file, so if we prepopulate it with hints it will make it easier! Good idea with the gitignore !